core/domain/draft_upgrade_services_test.py

```diff
@@ -1179,22 +1179,29 @@ def test_convert_states_v33_dict_to_v34_dict(self) -> None:
                 'property_name': 'written_translations',
                 'new_value': {
                     'translations_mapping': {
+                        # Here we use MyPy ignore because we are testing convert
+                        # function and in convert function we are working with
+                        # previous versions of the domain object and in previous
+                        # versions of the domain object there are some fields
+                        # (eg: html) that are discontinued in the latest domain
```

Comment:
Can you point to the latest domain object? I'm asking because I don't believe html is actually discontinued ... I thought it was still in use. Ditto everywhere else.

Reply:
https://github.com/oppia/oppia/blob/37ff4591fc7e57f273e54ac5a0226b8561086dbf/core/domain/state_domain.py#L1931

Reply:
Hm ok. I think these are wrong. Let's update them to the correct ones. I'm a bit surprised this code is still working tbh.

Ditto below.

@kevintab95 have there been any server errors related to this? (issues with converting drafts that have translations) I would guess it just results in a failed conversion?

Reply:
> I think these are wrong. Let's update them to the correct ones

I'm not sure to which field we should convert - I have sent an email to Kevin I hope he reply soon

Reply:
Written translations are no longer treated as a "state property" in either the CD or in the translation tab in the editor. We now use a dedicated change command that records the change in the content as well as the "data format" corresponding to the content (e.g. HTML, unicode, set of unicode, so on). That said, in the apply_change_list function, we still maintain support for the old CMD so either accepting suggestions or applying draft changes with the older "state property" CMD in the server would not result in an error.